backend/migrations/20210216132638_create-table-refreshTokens.ts

```diff
@@ -0,0 +1,16 @@
+import * as Knex from "knex"
+
+export async function up(knex: Knex): Promise<any> {
+  await knex.raw(`CREATE TABLE IF NOT EXISTS refreshTokens (
```

Comment:
I think this should be `refresh_tokens`, just to be consistent

Reply:
I should actually remove this since it's not in use in any capacity.